zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/Interpreter.java

```diff
@@ -156,7 +157,7 @@ public abstract InterpreterResult interpret(String st,
   @ZeppelinApi
   public List<InterpreterCompletion> completion(String buf, int cursor,
       InterpreterContext interpreterContext) throws InterpreterException {
-    return null;
+    return new ArrayList<>();
```

Comment:
do we need mutable List here? If not, can we use `Collections.emptyList()` ?

Reply:
Good point, will fix it